en/docs/develop/writing-a-custom-event-handler.md

```diff
@@ -47,8 +51,19 @@ The following list is a list of sample events.
 - `POST_UPDATE_USER_LIST_OF_ROLE`
 - `PRE_UPDATE_ROLE_LIST_OF_USER`
 - `POST_UPDATE_ROLE_LIST_OF_USER`
-- `UPDATE_GOVERNANCE_CONFIGURATION`
+- `POST_REVOKE_ACESS_TOKEN`
+- `POST_REVOKE_CODE`
+- `POST_REVOKE_ACESS_TOKEN_BY_ID`
+- `POST_REVOKE_CODE_BY_ID`
+- `POST_REFRESH_TOKEN`
+- `POST_ISSUE_CODE`
+- `POST_ISSUE_ACCESS_TOKEN`
+- `SESSION_CREATE`
+- `SESSION_UPDATE`
+- `SESSION_TERMINATE`
```

Comment:
There should be an event called SESSION_EXPIRE as well
@sachiniWettasinghe is this not included by default in the identity-event.properties file?

Reply:
This is added in the default.json file https://github.com/sachiniWettasinghe/carbon-identity-framework/blob/cb349df167dacbb511997d663a9d36a976f4df22/features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/org.wso2.carbon.identity.event.server.feature.default.json#L144